HELM-341: Documentation formatting cleanup (#484)
OpenNMS/opennms-helm: docs/modules/installation/pages/source.adoc

> = Install from Source

Reviewer: ```suggestion

= Install from Source
```

> = Installing From Source

Reviewer 2: ```suggestion
= Install from Source
```

> == Building

Reviewer 2: ```suggestion
== Build
```

== Build Docker Image from source

> The source code needs to be compiled as described in the "Building" section.

Reviewer 2: ```suggestion
You need to compile the source code as described in the "Build" section.
```
This should be a cross-reference, but again, we can update in a future Jira.
